src/mnode/mnode-pastel.cpp

```diff
@@ -1174,13 +1174,68 @@ CArtTradeTicket CArtTradeTicket::Create(std::string _sellTnxId, std::string _buy
     ticket.price = sellTicket->askedPrice;
 
     ticket.GenerateTimestamp();
-    
+
+    // In case it is nested it means that we have the artTnxId of the sell ticket
+    // available within the trade tickets.
+    // [0]: original registration ticket's txid
+    // [1]: copy number for a given art
+    std::vector<std::string> artRegTicket_TxId_Serial = CArtTradeTicket::GetArtRegTxIDAndSerialIfResoldNft(sellTicket->artTnxId);
+    if(artRegTicket_TxId_Serial[0].compare("") == 0)
```

Comment:
Should use empty() to make it easier to read.

Reply:
I better leave it like this. Because purpose is a comparison, and if we use empty it could be misleading in terms of the logic if someone reads freedcamp task. Since both gives same result, no issue here.